coverage_point_calculator/src/lib.rs

```diff
@@ -247,7 +247,6 @@ impl CoveragePoints {
             SpBoostedHexStatus::Eligible => self.coverage_points.boosted,
             SpBoostedHexStatus::WifiLocationScoreBelowThreshold(_) => dec!(0),
             SpBoostedHexStatus::AverageAssertedDistanceOverLimit(_) => dec!(0),
-            SpBoostedHexStatus::RadioThresholdNotMet => dec!(0),
```

Comment:
Should we also remove these comments?

- https://github.com/helium/oracles/blob/main/coverage_point_calculator/src/lib.rs#L52
- https://github.com/helium/oracles/blob/main/coverage_point_calculator/src/lib.rs#L24